cpp/models/lct_secir/infection_state.h

```diff
@@ -0,0 +1,247 @@
+/* 
+* Copyright (C) 2020-2024 MEmilio
+*
+* Authors: Lena Ploetzke
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef LCTSECIR_INFECTIONSTATE_H
+#define LCTSECIR_INFECTIONSTATE_H
+
+#include "memilio/utils/logging.h"
+#include <vector>
+
+namespace mio
+{
+namespace lsecir
+{
+
+/**
+ * @brief The InfectionStateBase enum describes the possible basic
+ * categories for the infection state of persons
+ */
+enum class InfectionStateBase
+{
+    Susceptible        = 0,
+    Exposed            = 1,
+    InfectedNoSymptoms = 2,
+    InfectedSymptoms   = 3,
+    InfectedSevere     = 4,
+    InfectedCritical   = 5,
+    Recovered          = 6,
+    Dead               = 7,
+    Count              = 8
+};
+
+/**
+ * @brief The InfectionTransition enum describes the possible
+ * transitions of the infectious state of persons.
+ */
+enum class InfectionTransition
+{
```

Comment:
Where do you need this?

Reply:
You are right, actually never, this only becomes relevant in issue #897 . I will delete this part for this PR.